lib/client.js

```
@@ -1,149 +1,160 @@
+      return store || null;
+    },
+    patchMessageBeforeSending: (message) => {
+      const requiresPatch = !!(
+        message.buttonsMessage ||
+        message.templateMessage ||
+        message.listMessage ||
+        message.scheduledCallCreationMessage ||
+        message.callLogMesssage
+      );
+      if (requiresPatch) {
+        message = {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)